filters/private/delaunator.hpp

```diff
@@ -261,15 +238,15 @@ Delaunator::Delaunator(std::vector<double> const& in_coords)
     ids.reserve(n);
 
     for (std::size_t i = 0; i < n; i++) {
-        const double x = coords[2 * i];
-        const double y = coords[2 * i + 1];
+      const double x = coords[2 * i];
```

Comment:
I wouldn't be opposed to a separate PR that cleans up any inconsistent code formatting, but the whitespace change in this PR detracts from the substantive changes you've made. It may be a little more clear, if you were to back these out.

Reply:
Certainly, will address

Reply:
Apropos of that, I have some general questions about the mechanics of the project (e.g. is there a mechanical way to achieve the desired style [like some set of flags to some pretty printer]?).  Where is the best place to ask those types of questions?

Reply:
Some of us will use either astyle or clang-format (I prefer the latter), both of which have style files in the root directory, but there is no standard. As much as anything, we generally like to keep a file internally consistent. Running either tool on this file _might_ introduce a ton of formatting changes. When that's done, it's typically performed separately from any functional changes for clarity.

The mailing list or Gitter would be reasonable places to ask such questions.

Reply:
Okay, many thanks

Reply:
Okay, I have reverted all changes to `delaunator.hpp` and instead re-declared the public interface in `filters/private/delaunator-decl.hpp`.  This is somewhat unattractive, but seems like it might be the best option for now?